codecov.yml

```diff
@@ -6,8 +6,14 @@ coverage:
       default: false
       python:
         paths:
-        - "src/sentry"
+        - "src/sentry/**/*.py"
```

Comment:
Does python use any other file extensions or is this a valid assumption to make @MaxBittker